frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/ToolJetDbOperations.jsx

```diff
@@ -337,9 +337,20 @@ const ToolJetDbOperations = ({ optionchanged, options, darkMode, isHorizontalLay
     ]
   );
 
+  const triggerTooljetDBStatus = (key) => {
+    if (window.public_config?.ENABLE_TOOLJET_DB !== 'true') {
+      toast('Tooljet database is not connected.', {
+        icon: '⚠️',
+      });
+    } else if (key === 'addTJDBTable') {
+      navigate(getPrivateRoute('database'));
+    }
+  };
+
   const fetchTables = async () => {
     const { error, data } = await tooljetDatabaseService.findAll(organizationId);
 
+    triggerTooljetDBStatus();
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/ToolJet/ToolJet/pull/10048#issuecomment-2161411807>#a4e3c7</a><span> - 06/11/2024, 06:54 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'triggerTooljetDBStatus' function is called inside 'fetchTables' without any arguments. This could lead to unexpected behavior since the function expects a 'key' parameter.
</div>
<div id="fix">
<b>Fix:</b> Ensure that 'triggerTooljetDBStatus' is called with the appropriate 'key' argument or modify the function to handle cases where 'key' is undefined.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -352,7 +352,7 @@
 
 -    triggerTooljetDBStatus();
 +    triggerTooljetDBStatus(undefined);
 
</code></pre>
</div>
</details>
</div>
<br>


</div>